.github/workflows/push-build.yaml

```diff
@@ -0,0 +1,26 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+jobs:
+  image-build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: build rook
+      run: |
+        # set VERSION to a dummy value since Jenkins normally sets it for us. Do this to make Helm happy and not fail with "Error: Invalid Semantic Version"
+        GOPATH=$(go env GOPATH) make clean && make -j$nproc VERSION=0 build
+        docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
+        docker images
+        docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:latest
+        docker push rook/ceph:latest
```

Comment:
Each master build currently pushes two tags: `rook/ceph:master` and something like `rook/ceph:v1.5.0.80.gaac0e5f` based on a git command. We don't use the `latest` tag. See the tags published [here](https://hub.docker.com/r/rook/ceph/tags/). We also push both amd64 and arm64 images, so this task will be a bit more complicated.

Reply:
1. First task can be using `${{ secrets.DOCKER_USER }}/back:${{ github.sha }}`?
2. for amd64, arm64 see this https://github.com/marketplace/actions/build-and-push-docker-images. I'm trying to understand how I can use this.